src/commands/server.cpp

```diff
@@ -21,7 +21,7 @@
 #include <string>
 #include <vector>
 
-extern volatile std::sig_atomic_t eoserv_sig_abort;
+volatile std::sig_atomic_t eoserv_sig_abort = false;
```

Comment:
We could specify this as `static` as well, right? (i.e., `static volatile`)

Reply:
From what I've been reading, static signals to the linker that a variable name is local to a single translation unit, which we don't want for this since it is referred to via extern in other places.

Although I do admit that my understanding of static in C++ is pretty fuzzy.